### PR TITLE
debug tempory view for test css

### PR DESCRIPTION
**Pull request by nibor51**

### CODE REVIEW

File: templates/part/index.html.twig [hunk 1]
```diff
@@ -3,8 +3,7 @@
 {% block title %}Part index{% endblock %}
 
 {% block body %}
-    <h2></h2>
-    <section class="nameMotor">
+<section class="nameMotor">
```
Comment (rocambille) [inline templates/part/index.html.twig]: fix indentation

File: templates/vehicle/partShow.html.twig [hunk 2]
```diff
@@ -5,19 +5,19 @@
...
     </section>
     <div class="flex parts">
         <h2>Temps d'utilisation total : {{ vehicle.0.usedHour }}H</h2>
         {% for part in parts %}
         <div class="namePart">
             <label for="{{ part.id }}">{{ part.name }}</label>
-            <progress value = "{{ vehicle.id.usedHour }}" max = "{{ part.lifespan }}"> </progress>
             <input type="checkbox" name="my-checkbox" id="{{ part.id }}">
         </div>
+            <progress value = "{{ vehicle.0.usedHour }}" max = "{{ part.lifespan }}"> </progress>
```
Comment (rocambille) [inline templates/vehicle/partShow.html.twig]: fix indentation

File: templates/vehicle/partShow.html.twig [hunk 3]
```diff
@@ -5,19 +5,19 @@
...
         {% endfor %}
-        <a class="buttonReplace" href="{{ path('part_edit') }}">Remplacer pièce</a>
-        <a class="buttonAddPiece" href="{{ path('part_new') }}">Ajouter pièce</a>
+        <a class="buttonReplace" href="">Remplacer pièce</a>
+        <a class="buttonAddPiece" href="">Ajouter pièce</a>
```
Comment (rocambille) [inline templates/vehicle/partShow.html.twig]: if you want "dummy" links, prefer href="#" (will not the reload the page)

**Comment by rocambille** [comment]
conflicts to resolve

**Comment by nibor51** [comment]
Done

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
sorry, I didn't click on the button "add your review" ^^'

File: templates/vehicle/partShow.html.twig (hunk 3 above)
Comment (JmJames51) [reply templates/vehicle/partShow.html.twig]: done

File: templates/vehicle/partShow.html.twig (hunk 2 above)
Comment (JmJames51) [reply templates/vehicle/partShow.html.twig]: done

File: templates/part/index.html.twig (hunk 1 above)
Comment (JmJames51) [reply templates/part/index.html.twig]: done

**Pull request merged**